src/sigmatcher/cli.py

```diff
@@ -161,7 +161,7 @@ def analyze(
     apk_hash = hashlib.sha256(apk.read_bytes()).hexdigest()
     unpacked_path = CACHE_DIR_PATH / apk_hash
     if not unpacked_path.exists():
-        subprocess.run([apktool, "decode", apk, "-f", "--output", unpacked_path.with_suffix(".tmp")])
+        subprocess.run([apktool, "decode", apk, "-r", "--no-assets", "-f", "--output", unpacked_path.with_suffix(".tmp")])
```

Comment:
Smart idea 😄 
My only nitpick is to change `-r` to `--no-res`

Reply:
I was wondering about that when I saw you using `decode` instead of "d" :) 

Will do in a while ;)